README.md

```diff
@@ -1,463 +1,161 @@
+- [x] 🏁 Developers can use course editor
+- [x] 🏁 Basic theme/branding
+- [x] 🏁 Course progress can be properly synchronised across devices
+- [x] 🏁 Course editor can be accessed by all course contributors
+- [x] 🏁 At least the basics of the Spanish course are ready
+- [x] 🏁 Most of the course building blocks are implemented
+- [x] 🏁 All of the basic building blocks of courses are implemented
+- [x] 🏁 Spaced repetition implemented
 
-* 🏁 Demo course content in Spanish for English speakers
-* 🏁 Some basic building blocks of course content implemented
-* 🏁 Developers can use course editor
-* 🏁 Basic theme/branding
-* 🏁 Course progress can be properly synchronised across devices
-* 🏁 Course editor can be accessed by all course contributors
-* 🏁 At least the basics of the Spanish course are ready
-* 🏁 Most of the course building blocks are implemented
-* 🏁 All of the basic building blocks of courses are implemented
-* 🏁 Spaced repetition implemented
-
-
-## Become a contributor
-
-
-<table>
-  <tr>
-    <td>
-      <table>
-        <tr>
-          <td>
-            <h3>🌍 UI translation</h3>
-          </td>
-        </tr>
-        <tr>
-          <td>
-            <ul>
-              <li>
-                <a href="https://poeditor.com/join/project?hash=jKiKfen2Ye">🔗 POEditor translation project!</a>
-              </li>
-              <li></li>
-            </ul>
-          </td>
-        </tr>
-      </table>
-    </td>
-    <td>
-      <table>
-        <tr>
-          <td>
-            <h3>📖 Course content</h3>
-          </td>
-        </tr>
-        <tr>
-          <td>
-            <ul>
-              <li><a href="https://github.com/kantord/LibreLingo/tree/main/docs/courses">Getting started</a></li>
-              <li></li>
-            </ul>
-          </td>
-        </tr>
-      </table>
-    </td>
-    <td>
-      <table>
-        <tr>
-          <td>
-                <h3>👩🏾‍💻 Development</h3>
-          </td>
-        </tr>
-        <tr>
-          <td>
-              <ul>
-                <li>
-                  <a href="https://github.com/kantord/LibreLingo/blob/main/docs/CONTRIBUTING.md">🌟 Getting started</a>
-                </li>
-                <li>
-                  <a href="https://librelingo.app/docs/">📜 Development documentation</a>
-                </li>
-              </ul>
-          </td>
-        </tr>
-      </table>
-    </td>
-  </tr>
-</table>
-
-You can help LibreLingo by testing it and submitting feature requests or bug reports: https://github.com/kantord/LibreLingo/issues/new. If you want to get in touch, you can use my contact details on [my GitHub profile](https://github.com/kantord).
-
-#### Confused? Have a question?
-
-There is a Matrix/FreeNode IRC channel `#LibreLingo`, instructions for joining:
 
-* Matrix: [`#LibreLingo`](https://riot.im/app/#/group/+librelingo:matrix.org) on `matrix.org`
-* IRC: connect to `irc.freenode.net` with your favourite client and join `#LibreLingo`
+## Roadmap
```

Comment:
perhaps for roadmaps we could use github projects? it's something that can be updated live without changing the readme file

Reply:
it's true that there's a clear advantage of having this in the readme. but I'm afraid that it will be out of sync. it would be nice to have a "single source of truth" for what the current project roadmap is.

Actually I'm not even sure if we need to have a _roadmap_ here, as opposed to simply the next main focus or main objective.